lib/proxymode.pm

```diff
@@ -127,4 +127,75 @@ sub redirect_serial {
     select_console "root-console";
     $self->reset_curr_serialdev();
 }
+
+sub get_consoledev {
```

Comment:
This seems very simillar to https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L97